adserver/tasks.py

```diff
@@ -588,3 +590,38 @@ def notify_of_publisher_changes(difference_threshold=0.25, min_views=10_000):
                             ),
                         },
                     )
+
+
+@app.task(time_limit=60 * 60 * 4)
+def archive_offers(day=None, delete=False):
```

Comment:
I figure switch this to `delete=True` after we deploy this and see how it goes for a few days.